astroid/tests/unittest_nodes.py

```diff
@@ -173,6 +173,26 @@ def test_slice_and_subscripts(self):
         ast = abuilder.string_build(code)
         self.assertEqual(ast.as_string(), code)
 
+    def test_operator_precedence(self):
+        with open(resources.find('data/operator_precedence.py')) as f:
+            for code in f.readlines():
```

Comment:
You can drop the `readlines` code, just iterate over the file with `for code in f`.